### PR TITLE
Update the help page

### PR DESCRIPTION
**Pull request by brianhlin**
It wasn't clear that any LHC issues should go through GGUS

### CODE REVIEW

File: docs/common/help.md [hunk 1]
```diff
@@ -53,4 +52,3 @@ your applications and requirements.
     * <http://www.uscms.org/uscms_at_work/physics/computing/grid/index.shtml>
     * CMS Hyper News: <https://hypernews.cern.ch/HyperNews/CMS/get/osg-tier3.html>
     * CMS Twiki: <https://twiki.cern.ch/twiki/bin/viewauth/CMS/USTier3Computing>
-
```
Comment (edquist) [inline docs/common/help.md]: Hating on that trailing blank line, i see...

Comment (brianhlin) [reply docs/common/help.md]: Haha, that wasn't on purpose. But yeah, screw those trailing blank lines.

Comment (sthapa) [reply docs/common/help.md]: Not sure if this is a bug in the preview but I see a :::console in the preview.  Is there a space or something missing which is preventing :::console from being interpreted?

Comment (brianhlin) [reply docs/common/help.md]: @sthapa The `:::console` (and ``````console) constructs get processed by MkDocs so it'll appear as-is in the GitHub preview

**Pull request merged**